### PR TITLE
Issue/refactor default data source

### PR DESCRIPTION
**Pull request by SergioEstevao**
This PR just moves the default data source from outside the picker. 

This is some preparatory work to make the implementation of a custom view for the picker easier.

How to test:

 - Run the demo app
 - And check the picker opens correctly.

### CODE REVIEW

File: Pod/Classes/WPMediaPickerViewController.m [hunk 1]
```diff
@@ -45,7 +45,7 @@ - (void)setupNavigationController
     vc.filter = self.filter;
     vc.allowMultipleSelection = self.allowMultipleSelection;
     if (!self.dataSource) {
-        self.dataSource = [self defaulDataSource];
+        self.dataSource = [WPPHAssetDataSource sharedInstance];
```
Comment (frosty) [inline Pod/Classes/WPMediaPickerViewController.m]: Does this need to be a singleton? Can we not just create a new one? I guess the issue is that `dataSource` is a weak reference?

Comment (SergioEstevao) [inline Pod/Classes/WPMediaPickerViewController.m]: Correct.

Comment (SergioEstevao) [inline Pod/Classes/WPMediaPickerViewController.m]: And keep in this this only applied has a default to allow the picker to work without extra configuration.

**Comment by frosty** [comment]
Sorry for the delay – :shipit:!

**Pull request merged**